src/schema/rules/files/raw/eyetrack.yaml

```diff
@@ -0,0 +1,22 @@
+---
+eyetrack:
+  suffixes:
+    - eyetrack
+  extensions:
+    - .tsv
+    - .json
+  datatypes:
+    - beh
+    - eeg
+    - func
+    - ieeg
+    - meg
+    - nirs
+    - perf
+    - pet
+  entities:
+    subject: required
+    session: optional
+    task: optional
+    acquisition: optional
+    run: optional
```

Comment:
it will probably be more appropriate to handle eyetrack files in the schema the same way that physio files are handled given that we want for an eyetrack file to have all the same entities as another datafile it is potentially associated to.

for example

for meg


```
sub-<label>/
    [ses-<label>/]
        meg/
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>][_recording-<label>]_eyetrack.tsv
```

for func

```
sub-<label>/
    [ses-<label>/]
        func/
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_rec-<label>][_dir-<label>][_run-<index>][_recording-<label>]_eyetrack.tsv
```

Reply:
+1 on harmonization with physio files!  With that note -- probably worth adding [StartTime](https://bids-specification--1128.org.readthedocs.build/en/1128/glossary.html#objects.metadata.StartTime) sidecar field, but we might need to adjust its description (or even "model") since ATM it talks about "Start time in seconds in relation to the start of acquisition of the first data sample in the corresponding **neural** dataset (negative values are allowed)." but it could be purely behavioral or some other non-neural modality I guess.  So we might need to make it somehow.

Given our recent quick chat with @Naubody: given we have `StartTime` and `SamplingFrequency`, we should provide minimal sufficient metadata to align eyetracking track to data (e.g. MRI) right?